modules/raycast/raycast_occlusion_cull.cpp

```diff
@@ -335,7 +335,9 @@ void RaycastOcclusionCull::Scenario::_update_dirty_instance(int p_idx, RID *p_in
 	int vertices_size = occ->vertices.size();
 
 	// Embree requires the last element to be readable by a 16-byte SSE load instruction, so we add padding to be safe.
-	occ_inst->xformed_vertices.resize(3 * vertices_size + 3);
+
+	occ_inst->xformed_vertices.resize(3 * vertices_size);
+	occ_inst->xformed_vertices.reserve(3 * vertices_size + 3);
```

Comment:
```suggestion
	occ_inst->xformed_vertices.reserve(3 * vertices_size + 3);
	occ_inst->xformed_vertices.resize(3 * vertices_size);
```
If we do end up sticking with this solution it would better to first `reserve()` and then `resize()` so that a situation doesn't come up where we do to memory allocations after each other.